### DISCUSSION TITLE
Two sessions token which leads to cloudfront rejecting the request

### DISCUSSION THREAD

**Discussion by thomaskvnze** [nextauthjs/next-auth#4810] [Help]
### Question 💬

I can see that next auth is setting two session tokens and at some point, the second one even grows in size such that total size of my header section exceeds maximum header section size limit of aws cloudfront. This way, next-auth is unusable for me. Can somebody explain to me why I have two session tokens and maybe why the token grows in size? I would assume that only one session token is important

### How to reproduce ☕️

`XXXX __Secure-next-auth.session-token.0=eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..ppqENOMp5Y1aeDGL.tzX1RWzY74L4bo7dWWkP4F6gCzDM3_irQIPETXg1Fc6ClmE2EmoClZKiSO3X53TccMQRnHHg-BtU8lZNX6jsSFndxX9UBPtTJOa8ViUVZbIsefOU_0lM_qnMWDGE_0K7nEeIdgtrz_PuEJQdp2Gq2DZ4tOKF6Iu26okKc7unm1yaEZBUVinPQEA5bogFKDJpDx-2-OFpkNH19txuvAnbMRpuo--1sT0xc0SLMg6KEe7ed7ArOMMp4uQkGaa2YRKvZ1ObbHxmxQ8PjhLKX7KZ1zn6Weusgj34oHGXzU-IontR1sOcQYo6cruKEjwgVQN4DVZ_VtJRJjN19neNK2KroKy588PyPBxQ1sIBYqupFKykS3KSOuSFO7_LlZEXiqzBV-CHWrv92W8pUfLHe8zngFoPE0WiGgieSmmWYYBRX098KkJy8gvB-731ORdztHqvRx_HcAN6YTPYl0Iya4xfFeWbxGW4VC68Y2cK6AUY4SuGtJ4X4pR5tIyTHT2ZbKgPRtfsAL0eLB73cq6xfxzigEB7_pEv_ws6KMB5mceDcHTP81U4OoAsJgLBM1L2980v6fdEJzAEtCNddMjCi_69uy6-d97E0573mFj7YPGOe8T8_yiel4btV985CcvbX_iJi09VX6KiYIpsy2zkOkUvjfyBJ9dgTpI8Sy1bnYBM7Dr1xZHwRbeTPge2pza4UX3GbnAvjGx7_9G9_c0MZvicI_2i1OmhVjtMXmTz5g8Hio7Q0ZA0Cxxrls9UkAwryAR8rTK2qUQg7l1pbCYpW9Y50oHtY8hjNOQNM9UYWimr_IxTf_nBqZYjbgtEUXWKnaCCz_VSJcoEBX0hoC3X8e81PuxjefJh-_5Crf097ypPit5M5_6gRQN5DXTuVH5qNqRqB4IPKBNkHigiaoHDwaZ9b8GxX6M-eEt8yW09XDW_SCJQuoHRHHs7WlEGT3zluOMcLTuHnAloPccLqDt39y_GafzPljXh7TNyKLw-AIw_hftTSixqvVO_S9NnP83HgO1eSfVfjRR8c-nm5xWm5PscoeghPzExmdw7tnX0Ef9t9Up6uDG_Sn9UV3R_Hn5paLIf7P-ClS2TGcWQQOWNmSuiyUsMI7Ri1XvlwpnrpCY1GAUO07TQiJGIQdWWrIyVYayVcWZLOvp2HJ-XI8O-yDMuZdd3InXCGkvagvPpaT1rqQw_tGy53Y0B8rpVbVyxPLDjjZ0sstf_r41Dp0BuhpNtIffUHgXZV_aQxVCRfFGTfwKE7O_LJQ4x2PWMyG25dNA9zByd1IvD8fSUIdslbL-Q6cTJj3AelVeJc4Qy9jNE-oVN72rXp0UUE16nvKPZ-ab6gUKdwYIOn3qLVGmg9-hrX9nUf8gPfgW8uNzI7DZhMxsLf37QTI0ipo962vpnpVB0myRWv1rwtyllM5W_nzGkUngtClvDHPE2yyI5bGEfN5FFbv_wp1gI-TQnoCg0_rF7n8MLreDzhRez6_FVxmVfZsAxoWNn0rbuRZq3264ne-J1DvDg02M0XKjErj71FE0fPchoKgTQzt_8r1Q27h_DLQ8H4S3LK709AqNJjF8gG6kbmzAnhe84VKAK9onqFipBTk-zQI9wgVPDF9ja9o8mGmrbWxRry6GpvbdNCUYFDbPg58J5iN4T0guaBKowvzhOrUm7LK1WftFdIAsnAiNz3awzVrKgvvohevbVfsLHV9LVjFRt5l-r00BTgXabWTAY89emfKKz06_6P4mCkv_kNTm1eZMfqbfFlYRMivqCbM7MhwXgfmSO9fowkD7lX2SpasjqsxNVgg3H1HYYKDoP5RzL5CSgA-1rcQrKBOXhMKASNCVDCULBTcOHxQDla_0O3UGnmarO1RbLduBJXrJ_rZKrfjn1UGghCWM0McvbVhF7MhSommMV6HglmA3TD1e5MMacdAlCoDRlOqM1C3kdsa017jWO5uok3S8322fni6ugOTDx0W63GxirXX9nNV2vJ69vZlmtOlfx8aMWSUHvzEvSdbjYb6C3A0t-EWRyxZ82TbFO-LKg0tc0G7P-HlWj2x0GSBhOxEe9MYLY4bmN8C5fOTKR_xrPmFo9YYlfEyqBWFQ4xeYdcjpLQjaLHEAkhbzOKgKYvUFpXmYqoH2cJ-WpJITfx9TdYLBysRrGi_flQQbiSVJfVi8mlveAZL0JvGsTv5RkWPhYzB2QvZlG5-M6LexKGQVCLksVyvlB041axu_lWTIOwPMNLQrrwzSnxXLV137mvAZDMhqQy-IwfXX-5KnAeBh9_vKAFiOHm5UgysTtLKm8qLAOl22TCq33KB7HBN8-OXlSRNH9-j7SZzRqCZ6Hvjkmv--vmaQVJvlGBhDAFFFKnUrvD4o8r5n3vb71bf2SBnZ4bxYk4HlVSUpl3pHx1rtWveS2nZ3pnqxnyiK65iiYg4rX5HwaMRDXVCT8Tyi8wg9YcJgaeQjGWxU1n7Vly4h81BpvuZRZYsFx3i10iMhRYBYQ9KNvaCKGWu87w4JhXCHrx1dwBZ6KcO8_M92KIdFRVV4R3s_pRYFcrRdegwKY4c4oD6RWo6I-EtNRJVPgWBS5WQ6fmipM7jI9SKjVFIG3E6d0y14VhleZfuj9RNockBzy7nQ7sBwy-DWVdN4tblO9Vfiqf_cQ_X0apaAs34UQz6a-icpLTiFMzNiw84yQFR_Cr7o6GjX1LmkhSKC4eBsdtEpVrBAWYFPoPwyVYqMoV0F5EOl4H9blevPYxtvaAIOWL5UhGyOglL4G5_7tbAS2ZnJJJa6VAxz2lLADST49tM38z_V2NgWVSgRlLjTknvLkW7vz8qJehbDuykzX4_RZxIBkB8FXjhEEFk122DP3TYL2Lw3KXRfiFgc2qw4MwgcsOjo78AMgt-BMWoa7SDFuVdStLnsJLUvD3IwprGatQgMskv1MnJQA_ZATeV9niz3rKTyp8YAJQR6XSUFmJvARAl_RPFCPRx6I510Tw23TSmKeEDVAQpSu90jhNE9DDNW8D7borFMLb-n6KEt-Lm-6YQHhyRfH8G0gS2-tDOR0EfZ9g7vuly0ZGIJdTKR7fkDPQr1osjUdPlieFQOf9gpPuseqbQ4yMK31ZgR4kkAWtrtJgYKUDFSiX0C0G39kx6cSl3Y0eFMyI_FeqALtjhW-42FYehgVZpRlzFpV5eVSAiQh_zTlZXuV5wdM2x0uBzWqTqgnRPOrYFSD3Nz9rIHLxuFYYliTFiU2eT-1F8pQTmJ3IbC8qdrCZDKNawA482_Jze449JJ4GVFG50qJHxEo7UAm_Xpbba80E38uC_qsD7i7JNQwFroGgtKQ23aVPbCYwNKMh4wgZWHEEFn6sV0ZZPM-QBSpj5pPRK2-Ld2IhabME7LQ_MZDR4ZRSZf4KWczUZffvd-0UavhsJ3c-0praHO2kA0SCdnFOk4aKac0xqzWkBgomKugG2YdUchAbi1rEd36f3EXqzNHBrmXVbz_0XoAxYeo4nq-loar3AYN8jPF6SKGPeF49iRbKhKx1xXMEmZSTkay_MtENameUPOF9aQXtuXP6jKdPjyMUoH_6SwHWfkyl8aMbEHPxBoQNYVuF_7ZLWCak5Y70Yb_gELDO1OqVbf1Ip1ljr-LC-cF093-XsALrqevzbP21pvVzhNgsYr0tYqTKNqLhH2raAPQ2nqhlODcpvvp2mxPbmyeRHU55KI3x8OLSJYoRjbP617OtItYbjJg_xE2GFhPOgJ0U1HifkeIaI7KWQtZB8JCi7kVe5Ti_9SNP1jqk6a92sBqt-zoklcBAYnuHvchNojo0-JQWdaZMF1o27MhM4yo3cJNYjfAwIjAY7y17vRGTTD4RxH4H2IJQbD8alrDSye7cf94WdhnfrcQP2qnaNwY_Tb; __Secure-next-auth.session-token.1=8x2H_jkaEcTESd4Yx1e14Yrgu7AxAA5HfnNFZNEfLtl-LXowbAGVgI4LODgllQYE4UJcgeiSJQy849QX3hQ.aHnq_O9g0H-1VqpQJ-uZrg`

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by EduardoFromTheFuture** [comment]
How come there is no reply on this issue?

I see that next-auth is creating two tokens for me when some users login but not all.

And I seem to be having some weird issues with requests not going through and I believe this is the cause.
